docs/en/rules/Azure.Redis.FirewallRuleCount.md

```diff
@@ -0,0 +1,27 @@
+---
+reviewed: 2022-09-22
+severity: Awareness
+pillar: Security
+category: Network security and containment
+resource: zure Cache for Redis
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Redis.FirewallRuleCount/
+---
+
+# Cleanup Redis cache firewall rules
+
+## Synopsis
+
+Determine if there is an excessive number of firewall rules for the Redis cache.
+
+## Description
+
+Azure Cache for Redis provides the functionality to create firewall rules, limiting the IP addresses that can access the resources. Normally, you want to limit the number of firewall rules.
+
+## RECOMMENDATION
+
+The Redis cache has more than ten (10) firewall rules. Some rules may not be needed.
+
```

Comment:
An example in ARM template/ Bicep of how customers can use an IP address range would be great.

For example:

```bicep
resource rule 'Microsoft.Cache/redis/firewallRules@2022-05-01' = {
  name: 'internal-app'
  parent: redis
  properties: {
    endIP: '10.0.1.128'
    startIP: '10.0.1.255'
  }
}
```